css/main.css

```diff
@@ -85,22 +85,21 @@ body {
     background-color: rgb(1,1,1,.7);
     background-position: center;
     box-shadow:inset 0 0 0 2000px rgba(0, 0, 0, 0.7);
-    background-size: 100%;
+    background-size: cover;
```

Comment:
what is the difference here? easier to change browser size without risking a side scroll?

Reply:
This CSS property changes how the background image is displayed when the container is taller than the image's aspect ratio. When set to `100%`, the width is equal to the width of the container, so black bars will show up on the top and bottom to maintain the aspect ratio. When set to `cover`, the background's scale is set to the minimum required to cover the entire div with the image, preventing borders on the sides.

Before: `background-size: 100%;`
<img width="247" alt="Screen Shot 2020-11-11 at 11 36 37 PM" src="https://user-images.githubusercontent.com/16531060/98896390-d4364780-2476-11eb-97df-a9bc2b8687b0.png">

After: `background-size: cover;`
<img width="247" alt="Screen Shot 2020-11-11 at 11 36 56 PM" src="https://user-images.githubusercontent.com/16531060/98896392-d4364780-2476-11eb-8d93-a39798020636.png">